hw/vendor/lowrisc_ibex/examples/sw/led/Makefile

```diff
@@ -10,7 +10,7 @@ ARCH = rv32imc
 # ARCH = rv32im # to disable compressed instructions
 SRCS = $(PROGRAM).c
 
-CC = /tools/riscv/bin/riscv32-unknown-elf-gcc
+CC ?= /tools/riscv/bin/riscv32-unknown-elf-gcc
```

Comment:
I think this change needs to go into [lowrisc/ibex](https://github.com/lowRISC/ibex) repo which gets 'vendored' into opentitan.

Reply:
Argh. I can look at updating this there then.